features/media-regenerate.feature

```diff
@@ -383,3 +383,216 @@ Feature: Regenerate WordPress attachments
       Success: Regenerated 1 of 1 images.
       """
     And the wp-content/uploads/white-150-square-150x150.jpg file should not exist
+
```

Comment:
There's tests for it in the current commit, for the non `--image_size` case at https://github.com/wp-cli/media-command/pull/9/commits/97cd95978b1dc4fdff08de24327825fb65419210#diff-7efe3459d2d56c929b2b92a4635e8fbfR241 lines L241-L307, and for `--image_size` at https://github.com/wp-cli/media-command/pull/9/commits/97cd95978b1dc4fdff08de24327825fb65419210#diff-7efe3459d2d56c929b2b92a4635e8fbfR567 lines L567-L636.